src/front-end/src/firebase-init.js

```diff
@@ -1,6 +1,5 @@
 import firebase from 'firebase/app';
 import 'firebase/auth';
-import 'firebase/database';
```

Comment:
Why delete this? This breaks functionality. 

The following functionality is now broken: when you signup as a mentor, the fields aren't saved to the realtime DB anymore. Please re-work your code and test again.

Reply:
done